### ISSUE TITLE
[cfn-lint] E8003: {'Fn::ImportValue': 'Stage'} is not of type 'string' when inside Fn::FindInMap

### ISSUE THREAD

**Issue by msplival** [aws-cloudformation/cfn-lint#3949]
### CloudFormation Lint Version

1.24.0

### What operating system are you using?

Ubuntu 24.04

### Describe the bug

I have a CloudFormation code snippet that looks like this:

```
Conditions:
  DeployEC2:
    Fn::Equals:
      - Fn::FindInMap:
        - PropertiesMap
        - DeployControllEC2
        - !Ref Stage
      - "true"
```

That cfn-lint has no problems with. 

But, for this I have to define a parametar called Stage. I'd like to import that value from the export of another stack, so I changed the snippet like this:

```
Conditions:
  DeployEC2:
    Fn::Equals:
      - Fn::FindInMap:
        - PropertiesMap
        - DeployControllEC2
        - !ImportValue Stage
      - "true"
```

Now I'm getting the error:

`[cfn-lint] E8003: {'Fn::ImportValue': 'Stage'} is not of type 'string'`

Why is this happening? My export, indeed, is a type of string - and !ImportValue indeed does return a string. 

Is this a problem with the way !ImportValue is used within FindInMap? 

### Expected behavior

I'd expect cfn-lint not to complain here, because !ImportValue indeed does return a string.

### Reproduction template

```yaml
AWSTemplateFormatVersion: '2010-09-09'

Parameters:
  Stage:
    Type: String
    Default: dev

Mappings:
  PropertiesMap:
    DeployControllEC2:
      dev: true
      prod: true

Conditions:
  DeployEC2:
    Fn::Equals:
      - Fn::FindInMap:
        - PropertiesMap
        - DeployControllEC2
        - !ImportValue Stage
      - "true"
```

**Comment by kddejong** [comment]
This is representative of what validate template will provide you as an error.
```
➜  aws cloudformation validate-template --template-body file://i3949.yaml

An error occurred (ValidationError) when calling the ValidateTemplate operation: Template error: every Fn::FindInMap object requires three parameters, the map name, map key and the attribute for return value
```

Also the [docs](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference-findinmap.html#w32aac23c24c26c13) say the only supported functions are `Ref` and `Fn::FindInMap`

**Comment by msplival** [comment]
@kddejong eh, thnx! It was staring right at me :) 

So, the only way to get the export from another stack is via parametar, but then I need to use external (out-of-cloudformation) scripting to get the export.

But I guess it's not cfn-lint's issue. 

Thank you again for pointing me to right place!
